src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -38,7 +39,7 @@ namespace Microsoft.ML.Transforms
     /// <include file='doc.xml' path='doc/members/member[@name="TensorflowTransform"]/*' />
     public sealed class TensorFlowTransform : ITransformer, ICanSaveModel
     {
-        public sealed class Arguments : TransformInputBase
+        public class Arguments : TransformInputBase
```

Comment:
>public class Arguments [](start = 8, length = 22)

Can you make it abstract? Also I would suggest to call it ArgumentsBase rather than Arguments, because it would be confusing to have Arguments class which is actually an class with params and this class.

Reply:
It can not be made abstract because `Arguments` object is instantiated for scoring scenario and `TrainingArguments` is instantiated for training scenario.

---
In reply to: [221764289](https://github.com/dotnet/machinelearning/pull/1063#discussion_r221764289) [](ancestors = 221764289)

Reply:
Why it can't remain sealed if you don't use it as base class?
Also maybe rename it into ScoringArguments?

---
In reply to: [221768356](https://github.com/dotnet/machinelearning/pull/1063#discussion_r221768356) [](ancestors = 221768356,221764289)